build/nuget/Uno.WinUI.nuspec

```diff
@@ -654,7 +654,9 @@
 		<file src="..\..\src\SourceGenerators\Uno.UI.Tasks\Bin\Release\*.dll" target="buildTransitive\Uno.UI.Tasks" />
 		<file src="..\..\src\SourceGenerators\Uno.UI.Tasks\Bin\Release\*.pdb" target="buildTransitive\Uno.UI.Tasks" />
 
-		<file src="..\..\src\SourceGenerators\Uno.UI.Tasks\external\linker\*.*" target="tools\linker" />
+		<!-- Without tools directory, we'll not get the PkgUno_WinUI property generated by NuGet's RestoreTask. -->
+		<!-- Currently, we're using PkgUno_WinUI in few places. In case we replaced its usage with something else, we can remove this empty tools folder -->
```

Comment:
```suggestion
		<!-- Currently, we're using PkgUno_WinUI in few places. In case we replace its usage with something else, we can remove this empty tools folder -->
```